src/libs/IOUUtils.js

```diff
@@ -63,7 +64,77 @@ function updateIOUOwnerAndTotal(iouReport, actorEmail, amount, currency, type =
     return iouReportUpdate;
 }
 
+/**
+ * Returns the list of IOU actions depnding on the type and whehter or not they are pending.
+ * Used below so that we can decide if an IOU report is pending currency conversion.
+ *
+ * @param {Array} reportIOUActions
+ * @param {Object} iouReport
+ * @param {String} type - iouReportAction type. Can be oneOf(create, decline, cancel, pay, split)
+ * @param {String} pendingAction
+ * @param {Boolean} filterRequestsInDifferentCurrency
+ *
+ * @returns {Array}
+ */
+function getIOUReportActions(reportIOUActions, iouReport, type = '', pendingAction = '', filterRequestsInDifferentCurrency = false) {
```

Comment:
Why name the first param as `reportIOUActions` when it can be any report action? because that looks confusing.